utils/imodels-client-test-utils/src/assertions/NodeOnlyAssertions.ts

```diff
@@ -138,6 +138,24 @@ export async function assertDownloadedChangeset(params: {
   expect(fs.statSync(params.actualChangeset.filePath).size).to.equal(fs.statSync(params.expectedTestChangesetFile.filePath).size);
 }
 
+export async function assertChangesetGroup(params: {
+  actualChangesetGroup: ChangesetGroup;
+  expectedChangesetGroupProperties: ChangesetGroupPropertiesForCreate & Partial<ChangesetGroupPropertiesForUpdate>;
+}): Promise<void> {
+  expect(params.actualChangesetGroup.id).to.not.be.empty;
+  expect(params.actualChangesetGroup.description).to.equal(params.expectedChangesetGroupProperties.description);
+  expect(params.actualChangesetGroup.creatorId).to.exist;
```

Comment:
`id` is asserted with `to.not.be.empty`, `creatorId` is asserted with `to.exist`. We should unify how strings are asserted, I personally think `to.not.be.empty` reads a bit better